models/src/main/scala/coop/rchain/models/BlockHash.scala

```diff
@@ -5,5 +5,6 @@ import com.google.protobuf.ByteString
 object BlockHash {
   type BlockHash = ByteString
 
-  val Length = 32
+  val Length    = 32 // length in bytes
+  val HexLength = 64 // each byte is 2 chars
```

Comment:
Your comment is correct but code duplicate the information about length.
`val HexLength = Length * 2`